apps/src/blockly/addons/functionEditor.js

```diff
@@ -44,6 +44,7 @@ export default class FunctionEditor {
 
     // Customize auto-populated Functions toolbox category.
     this.editorWorkspace = Blockly.blockly_.inject(modalEditor, {
+      ...Blockly.getMainWorkspace().options,
```

Comment:
Was there something specific that you wanted to pull in here? Is this what makes unconnected blocks disabled?

I left this out because someone could add an option to the main workspace that we don't want to be propagated to the function editor. Is it worth adding a comment to the spot where we initialize the main workspace options calling out that anything added there will also apply to the function editor unless explicitly overridden?

Reply:
There was some styling around unconnected blocks I needed. I can try to be more specific and find the setting rather than pulling in everything.

Reply:
I just needed to pull in the renderer, updating!